src/sbt-test/ensime-sbt/scalariform/build.sbt

```diff
@@ -0,0 +1,17 @@
+ivyLoggingLevel := UpdateLogging.Quiet
+
+import scalariform.formatter.preferences._
+import com.typesafe.sbt.SbtScalariform
+import com.typesafe.sbt.SbtScalariform.ScalariformKeys
+
+SbtScalariform.scalariformSettings
+
+ScalariformKeys.preferences := FormattingPreferences
+  .setPreference(DoubleIndentClassDeclaration, true)
+  .setPreference(DanglingCloseParenthesis, Preserve)
+  .setPreference(MultilineScaladocCommentsStartOnFirstLine, false)
+  .setPreference(PlaceScaladocAsterisksBeneathSecondAsterisk, true)
+  .setPreference(AlignSingleLineCaseStatements.MaxArrowIndent, 20)
+
+scalaVersion := "2.11.7"
```

Comment:
oh darn, I didn't realise we did that. ok, leave it then, or update to 2.11.8 if you're really brave.